src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -101,8 +103,7 @@ public partial class UpdateDurableFunction
 
             if (instanceWatermarks.Count > 0)
             {
-                _updateMeter.UpdatedInstances.Add(instanceWatermarks.Count,
-                    new KeyValuePair<string, object>("ExecutionId", context.NewGuid()));
+                replaySafeCounter.Add(instanceWatermarks.Count);
```

Comment:
This is great, how do we know for a particular orchestration if the counter is called once or twice?

Reply:
This is about replay -- for when the orchestration is rebuilding state -- not necessarily for failure. There is no signal for whether the orchestration restarted and already sent a counter that is not part of a replay